### DISCUSSION TITLE
Code Announcements for Users

### DISCUSSION THREAD

**Discussion by kaiwu-astro** [nbody6ppgpu/Nbody6PPGPU-beijing#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by spurzem** [comment]
NOTICE about broken backward compatibilty for stardisk-dev version: please note that in accretion.f for R_TIDAL = 0.22*RACCR the model-dependent factor 0.22 has been TAKEN AWAY!!  It means that the input parameter RACCR is now directly the tidal radius for the reference star (currently one solar mass). It means also if you want to compare an older run with a new run from now on, you need to change RACCR if you want to have same R_TIDAL.

**Reply by kaiwu-astro** [reply]
For users: 
- this change happens after commit `bc6dbc6` in stardisk-dev branch. 
- If you want to use the old version, after `git clone; git switch stardisk-dev`, do: `git reset bc6dbc6`

**Comment by spurzem** [comment]
another broken backwards compatibility
- commit: https://github.com/nbody6ppgpu/Nbody6PPGPU-beijing/commit/5539e77872cde1a7c3a01c5b46eb8d76750d3d2a
- affected versions: all 
- where: routine `hrplot.F`
- what: the output of RI in sev and bev files, radial position in the cluster, will be now in PC (multiplied by RBAR) rather than in units of core radii. At the same time also more output was added (tm: main sequence lifetime; mc, me, rc, re: core and envelope mass and radii). This is backward compatible, additional items were just added to the line. 

A corresponsing update of the HDF5 output is still work in progress, not finished.
